services/download/src/test/java/app/coronawarn/server/services/download/FederationBatchProcessorTest.java

```diff
@@ -37,7 +38,10 @@
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
 import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
```

Comment:
wouldn't an `import static org.mockito.Mockito.verify` make this change a lot smaller?